firefox/guide/firefox/firefox_preferences-background_data/oval/firefox.xml

```diff
@@ -0,0 +1,25 @@
+<def-group>
+  <definition class="compliance" id="firefox_preferences-background_data"  version="1">
+    <metadata>
+      <title>Background submission of information to Mozilla must be disabled.</title>
```

Comment:
Let's bring this inline with the other rules. Please make the title:
`Disable Background Information Submission`

Reply:
Edited the title accordingly